libs/ui/switch/brain/src/lib/brn-switch-ng-model.spec.ts

```diff
@@ -51,11 +51,9 @@ describe('BrnSwitchComponentNgModelIntegration', () => {
 		expect(container.fixture.componentInstance.airplaneMode).toBe(true);
 	});
 
-	// TODO - why does this test not fail? it should be setup(false, true) but setup(false, false) still passes
```

Comment:
this TODO is not resolved? 
the problem is, that the test passes, although the setup does not set it to disabled. 
I am not sure why this is the case, maybe you have? ;)

Reply:
I checked it once again. I though it was resolved by missing unsubscribe, but it obviously was not. Missing unsubcribe solved different issue...